python/fledge/services/core/api/audit.py

```diff
@@ -114,14 +116,22 @@ async def create_audit_entry(request):
                    'severity': severity,
                    'details': details
                    }
-        return web.json_response(message)
     except AttributeError as e:
         # Return error for wrong severity method
         err_msg = "severity type {} is not supported".format(severity)
         _logger.error("Error in create_audit_entry(): %s | %s", err_msg, str(e))
-        raise web.HTTPNotFound(reason=err_msg)
+        raise web.HTTPNotFound(reason=err_msg, body=json.dumps({"error": err_msg}))
+    except StorageServerError as ex:
+        if int(ex.code) in range(400, 500):
+            err_msg = 'Audit entry cannot be logged. {}'.format(ex.error['message'])
+            raise web.HTTPBadRequest(body=json.dumps({"error": err_msg}))
```

Comment:
Add 'reason' consistency wise across API & for the key KV change (same as above)